pydatalab/pydatalab/apps/chat/blocks.py

```diff
@@ -10,14 +11,14 @@
 from pydatalab.utils import CustomJSONEncoder
 
 __all__ = "ChatBlock"
-MODEL = "gpt-3.5-turbo-0613"
-MAX_CONTEXT_SIZE = 4097
 
 
 class ChatBlock(DataBlock):
     blocktype = "chat"
     description = "Virtual assistant"
     accepted_file_extensions: Sequence[str] = []
+    ChatClient = None
```

Comment:
Should import `langchain.chat_models.BaseChatModel` above somewhere then use this for the type. Also rename the attribute to `chat_client`.

```suggestion
    chat_client: BaseChatModel | None = None
```